generate_pipeline.py

```diff
@@ -0,0 +1,57 @@
+from jinja2 import Template
+from pathlib import Path
+
+ingests = []
+
+ingest_dirs = [dir.parent for dir in Path("./monarch_ingest").rglob("metadata.yaml")]
+
+for ingest_dir in ingest_dirs:
+    source_ingests = [ingest_file for ingest_file in list(ingest_dir.rglob("*.yaml"))
+                      if not ingest_file.match("metadata.yaml")]
+    for ingest in source_ingests:
+        ingests.append({
+            'source':str(ingest.parent.stem),
+            'source_file':str(ingest.name),
+            'source_file_stem': str(ingest.stem),
+            'ingest_method': 'ingest_' + str(ingest.parent.stem) + '_' + str(ingest.stem) + '()'
+        })
+
+ingest_methods = ', '.join([ingest['ingest_method'] for ingest in ingests])
+
+
+template = """
+import os
+from dagster import ModeDefinition, InputDefinition, Nothing, default_executors, multiprocess_executor, fs_io_manager, pipeline, solid, composite_solid
+from koza.cli_runner import transform_source
```

Comment:
making a note here that we should move transform_source out of cli_runner since it has some utility outside of a cli (or we can change of cli_runner back to koza_runner)